crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -18,6 +18,27 @@ pub struct Schedule {
     last_initialize_generation: usize,
 }
 
+impl fmt::Debug for Schedule {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        writeln!(f, "Schedule {{")?;
+
+        let stages = self
+            .stage_order
+            .iter()
+            .map(|s| (s, self.stages[s].iter().map(|s| (s.name(), s.id()))));
+
+        for (stage, syss) in stages {
+            writeln!(f, "\tStage \"{}\"", stage)?;
```

Comment:
The default std debug impls use four spaces for each indentation. Lets do that here for consistency.